src/staking/block_validator.cpp

```diff
@@ -156,10 +156,19 @@ class BlockValidatorImpl : public AbstractBlockValidator {
     if (block_header.hashPrevBlock != *previous_block.phashBlock) {
       result.AddError(Error::PREVIOUS_BLOCK_DOESNT_MATCH);
     }
-    if (block_header.GetBlockTime() <= previous_block.GetMedianTimePast()) {
+
+    const int64_t block_time = block_header.GetBlockTime();
+
+    if (block_time < previous_block.GetMedianTimePast()) {
```

Comment:
nit: no need of new line between `block_time` declaration and its guard-check as after new line I tend to expect a new logical block. See the example below with `on_demand` where we don't separate declaration with the `if` check

Reply:
I think this is deferrable to personal taste. The new line here for me is justified by the fact that the variable is used by many if blocks after, where i.e. `on_demand` just in the following.